Delicate-Butterflies/Bangazon-Site
# Seeders jl mb

Related Issue: #38 

Description:
This should setup seeders to that all the npm db scripts shouldn't have issues with the seeder part.

PR Testing Steps:
- "npm run dbrb" or "npm teardown" first if there is an issue
- check data in psql
- make sure login and register still works
